inductiva/simulators/fenicsx.py

```diff
@@ -21,6 +21,7 @@ def run(
         global_refinement_meshing_factor: float = 1.0,
         local_refinement_meshing_factor: float = 1.0,
         smoothing_meshing_parameter: float = 10.0,
+        mesh_element_degree: int = 1,
```

Comment:
What happens if I put 0?

Reply:
When attempting to run the simulation with fencisx, the following error occurs: "Traceback (most recent call last):
  File "/scripts/elastic_case/elastic_case.py", line 59, in <module>
    app.run(main)
  File "/usr/local/lib/python3.10/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.10/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/scripts/elastic_case/elastic_case.py", line 48, in main
    solver.solve()
  File "/scripts/elastic_case/linear_elasticity.py", line 688, in solve
    self.v_func_space = dolfinx.fem.VectorFunctionSpace(
  File "/usr/local/dolfinx-real/lib/python3.10/dist-packages/dolfinx/fem/function.py", line 605, in VectorFunctionSpace
    ufl_element = basix.ufl_wrapper.create_vector_element(
  File "/usr/local/lib/python3.10/dist-packages/basix/ufl_wrapper.py", line 1294, in create_vector_element
    e = create_element(family, cell, degree, lagrange_variant, dpc_variant, discontinuous, gdim)
  File "/usr/local/lib/python3.10/dist-packages/basix/ufl_wrapper.py", line 1268, in create_element
    e = _basix.create_element(family, cell, degree, lagrange_variant, dpc_variant, discontinuous)
RuntimeError: Cannot create a continuous order 0 Lagrange basis function"

Because the order of the element must be equal to or higher than 1.

Reply:
Alright, but that needs to go to the backend. If you want to fail, fail fast.

Reply:
I agree; I will incorporate this into the open pull request (https://github.com/inductiva/inductiva-web-api/pull/473) in the backend. Thanks, Ivan!